src/api/environment.cc

```diff
@@ -528,17 +528,15 @@ MaybeLocal<Value> LoadEnvironment(
   return StartExecution(env, cb);
 }
 
-MaybeLocal<Value> LoadEnvironment(
-    Environment* env,
-    const char* main_script_source_utf8) {
-  CHECK_NOT_NULL(main_script_source_utf8);
+MaybeLocal<Value> LoadEnvironment(Environment* env,
+                                  std::string_view main_script_source_utf8) {
+  CHECK_NOT_NULL(main_script_source_utf8.data());
   return LoadEnvironment(
       env, [&](const StartExecutionCallbackInfo& info) -> MaybeLocal<Value> {
-        std::string name = "embedder_main_" + std::to_string(env->thread_id());
```

Comment:
If we remove the `embedder_main_` part here, doesn't all occurrences of `embedder_main_` in `src/node_builtins.cc` like https://github.com/nodejs/node/blob/8d573d9985f06909fac4bcc3fc1ef7168c264e46/src/node_builtins.cc#L189 also require some change?